src/utils/LazyLoader.js

```diff
@@ -1,24 +1,28 @@
-export default {
-    datePickerRange: () =>
-        import(/* webpackChunkName: "datepicker" */ '../fragments/DatePickerRange.react'),
-    datePickerSingle: () =>
-        import(/* webpackChunkName: "datepicker" */ '../fragments/DatePickerSingle.react'),
-    dropdown: () =>
-        import(/* webpackChunkName: "dropdown" */ '../fragments/Dropdown.react'),
-    graph: () =>
-        import(/* webpackChunkName: "graph" */ '../fragments/Graph.react'),
-    markdown: () =>
-        import(/* webpackChunkName: "markdown" */ '../fragments/Markdown.react'),
-    plotly: () =>
-        Promise.resolve(
-            window.Plotly ||
-                import(/* webpackChunkName: "plotlyjs" */ 'plotly.js').then(
-                    ({default: Plotly}) => {
-                        window.Plotly = Plotly;
-                        return Plotly;
-                    }
-                )
-        ),
-    upload: () =>
-        import(/* webpackChunkName: "upload" */ '../fragments/Upload.react'),
-};
+export const datePickerRang = () =>
```

Comment:
rework the lazy loader so that it can be tree-shaken.. so.. not all on the same object 😄